Assets/MixedRealityToolkit.Examples/Demos/SpatialAwareness/Scripts/DemoSpatialMeshHandler.cs

```diff
@@ -1,18 +1,17 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Core.Definitions.SpatialAwarenessSystem;
-using Microsoft.MixedReality.Toolkit.Core.EventDatum.SpatialAwarenessSystem;
-using Microsoft.MixedReality.Toolkit.Core.Interfaces.SpatialAwarenessSystem.Handlers;
-using Microsoft.MixedReality.Toolkit.Core.Services;
-using Microsoft.MixedReality.Toolkit.Core.Utilities.Async;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.WindowsMixedReality.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.WindowsMixedReality.SpatialAwareness.Handlers;
+using Microsoft.MixedReality.Toolkit.Utilities;
 using System.Collections.Generic;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.Examples.Demos
 {
     /// <summary>
-    /// This class is an example of the <see cref="Microsoft.MixedReality.Toolkit.Core.Interfaces.SpatialAwarenessSystem.Handlers.IMixedRealitySpatialAwarenessObservationHandler{T}"/> interface. It keeps track
+    /// This class is an example of the <see cref="Microsoft.MixedReality.Toolkit.WindowsMixedReality.SpatialAwareness.Handlers.IMixedRealitySpatialAwarenessObservationHandler{T}"/> interface. It keeps track
```

Comment:
```suggestion
    /// This class is an example of the <see cref="Microsoft.MixedReality.Toolkit.SpatialAwareness.IMixedRealitySpatialAwarenessObservationHandler{T}"/> interface. It keeps track
```

Reply:
I personally like the condensed version, but I can change it if you like?

---
In reply to: [267587565](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267587565) [](ancestors = 267587565)